openrelik_worker_common/mount_utils.py

```diff
@@ -96,6 +103,24 @@ def _losetup(self) -> str:
 
         return blkdevice
 
+    def _required_tools_available(self) -> tuple[bool, str]:
+        """Check if required cli tools are available.
+
+        Returns:
+            tuple: tuple of return bool and error message
+        """
+        errmsg = ""
+        ok = True
+
+        tools = ["lsblk", "blkid", "mount"]
+
+        for tool in tools:
+            if not which(tool):
+                errmsg += f" {tool} "
+                ok = False
+
+        return (ok, errmsg)
```

Comment:
You can simplify this a bit, something like:
```suggestion
    tools = ["lsblk", "blkid", "mount"]
    missing_tools = [tool for tool in tools if not shutil.which(tool)]
    
    if missing_tools:
        error_message = f"Missing required tools: {' '.join(missing_tools)}"
        return False, error_message

    return True    
```

Also consider raising an exception instead or returning False seeing this would be a unrecoverable state.

```
class MissingToolsError(Exception):
    pass

if missing_tools:
    raise MissingToolsError(f"Missing required tools: {' '.join(missing_tools)}")
    
...
```